controllers/topolvm_controller.go

```diff
@@ -96,16 +98,14 @@ func (c topolvmController) ensureDeleted(r *LVMClusterReconciler, ctx context.Co
 	return nil
 }
 
-func getControllerDeployment(namespace string, initImage string, topoLVMLeaderElectionPassthrough v1.LeaderElection) *appsv1.Deployment {
+func getControllerDeployment(namespace string, topoLVMLeaderElectionPassthrough v1.LeaderElection) *appsv1.Deployment {
 	// Topolvm CSI Controller Deployment
 	var replicas int32 = 1
 	volumes := []corev1.Volume{
 		{Name: "socket-dir", VolumeSource: corev1.VolumeSource{EmptyDir: &corev1.EmptyDirVolumeSource{}}},
-		{Name: "certs", VolumeSource: corev1.VolumeSource{EmptyDir: &corev1.EmptyDirVolumeSource{}}},
-	}
-
-	initContainers := []corev1.Container{
-		initContainer(initImage),
+		// TopoLVM needs the certs for webhooks that are not deployed in LVMS, but there is no option to disable them from the controller webhook server.
```

Comment:
Why dont we deploy the webhooks for TopoLVM? Maybe also an oversight on TopoLVM side, if we dont use any webhooks we shouldnt need to mount them right? Maybe worth a PR on TopoLVM?

Reply:
Yes, I agree this is worth a PR on TopoLVM. I was going to propose this as an interim solution, but I guess this PR can wait until we improve this part on TopoLVM.